actions/bump-version/action.yaml

```diff
@@ -1,12 +1,12 @@
 name: 'Bump version'
-description: 'This action bumps package versions based on the repository's commit messages written according to the Conventional Commits specification.'
+description: "This action bumps package versions based on the repository's commit messages written according to the Conventional Commits specification."
```

Comment:
This is one of the problematic lines. Because it contained a single quote in the string, it broke the GitHub YAML parser